lib/parsers.c

```diff
@@ -185,17 +185,24 @@ static int issue_char(struct libwebsocket *wsi, unsigned char c)
 		return -1;
 	}
 
-	if( wsi->u.hdr.ah->frags[wsi->u.hdr.ah->next_frag_index].len >= 
-		wsi->u.hdr.current_token_limit) {
-		lwsl_warn("header %i exceeds limit\n", wsi->u.hdr.parser_state);
-		return 1;
+	unsigned short frag_len = \
```

Comment:
Please declare any variables at the top of the scope to be C89 compatible (Microsoft compiler only support C89)